nucleus/model.py

```diff
@@ -24,6 +24,9 @@ def __init__(
         self.metadata = metadata
         self._client = client
 
+    def __repr__(self):
+        return f'Model({self.id}, {self.name}, {self.reference_id}, {self.metadata}, {self._client})'
```

Comment:
__repr__ implementations should include argument names as kwargs so that the output results are readable.

```python
return f'Model(model_id={self.id}, name={self.name}, reference_id={self.reference_id}, metadata={self.metadata}, client={self._client})'
```